MiniTwitch.Helix/Internal/HelixResultFactory.cs

```diff
@@ -82,6 +86,7 @@ internal static class HelixResultFactory
         (HttpResponseMessage response, TimeSpan elapsed) = await client.RequestAsync(request, cancellationToken);
 
         var responseJson = await response.Content.ReadFromJsonAsync<JsonElement>(HelixApiClient.SerializerOptions, cancellationToken);
+        response.Dispose();
```

Comment:
Worth moving this to a `try`/`finally` or a plain old `using` scope in case the line above throws?

I don't remember the disposal situations for `HttpResponseMessage` so it may not actually matter in practice.

Reply:
completely overlooked this, thanks